.idea/modules.xml

```diff
@@ -2,56 +2,56 @@
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-pinpoint-test/aws-android-sdk-pinpoint-test.iml" filepath="$PROJECT_DIR$/aws-android-sdk-pinpoint-test/aws-android-sdk-pinpoint-test.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-polly/aws-android-sdk-polly.iml" filepath="$PROJECT_DIR$/aws-android-sdk-polly/aws-android-sdk-polly.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-rekognition/aws-android-sdk-rekognition.iml" filepath="$PROJECT_DIR$/aws-android-sdk-rekognition/aws-android-sdk-rekognition.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-s3/aws-android-sdk-s3.iml" filepath="$PROJECT_DIR$/aws-android-sdk-s3/aws-android-sdk-s3.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-s3-test/aws-android-sdk-s3-test.iml" filepath="$PROJECT_DIR$/aws-android-sdk-s3-test/aws-android-sdk-s3-test.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-sagemaker-runtime/aws-android-sdk-sagemaker-runtime.iml" filepath="$PROJECT_DIR$/aws-android-sdk-sagemaker-runtime/aws-android-sdk-sagemaker-runtime.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-sdb/aws-android-sdk-sdb.iml" filepath="$PROJECT_DIR$/aws-android-sdk-sdb/aws-android-sdk-sdb.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-ses/aws-android-sdk-ses.iml" filepath="$PROJECT_DIR$/aws-android-sdk-ses/aws-android-sdk-ses.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-sns/aws-android-sdk-sns.iml" filepath="$PROJECT_DIR$/aws-android-sdk-sns/aws-android-sdk-sns.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-sqs/aws-android-sdk-sqs.iml" filepath="$PROJECT_DIR$/aws-android-sdk-sqs/aws-android-sdk-sqs.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-testutils/aws-android-sdk-testutils.iml" filepath="$PROJECT_DIR$/aws-android-sdk-testutils/aws-android-sdk-testutils.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-textract/aws-android-sdk-textract.iml" filepath="$PROJECT_DIR$/aws-android-sdk-textract/aws-android-sdk-textract.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-transcribe/aws-android-sdk-transcribe.iml" filepath="$PROJECT_DIR$/aws-android-sdk-transcribe/aws-android-sdk-transcribe.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-translate/aws-android-sdk-translate.iml" filepath="$PROJECT_DIR$/aws-android-sdk-translate/aws-android-sdk-translate.iml" />
-      <module fileurl="file://$PROJECT_DIR$/aws-sdk-android.iml" filepath="$PROJECT_DIR$/aws-sdk-android.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/modules/aws-android-sdk-apigateway-core/aws-android-sdk-apigateway-core.iml" filepath="$PROJECT_DIR$/.idea/modules/aws-android-sdk-apigateway-core/aws-android-sdk-apigateway-core.iml" group="aws-sdk-android/aws-android-sdk-apigateway-core" />
```

Comment:
Is it worth tracking module.xml file via Git?

Reply:
I Googled this topic a bit. People say just not to include any `.idea/*` files, so I just removed the entire directory.